Mathlib/Analysis/SpecialFunctions/Log/ENNRealLog.lean

```diff
@@ -77,39 +77,20 @@ section Monotonicity
 theorem log_strictMono : StrictMono log := by
   intro x y h
   unfold log
-  rcases ENNReal.trichotomy x with (rfl | rfl | x_real)
-  · rcases ENNReal.trichotomy y with (rfl | rfl | y_real)
-    · exfalso; exact lt_irrefl 0 h
-    · simp
-    · simp [(ENNReal.toReal_pos_iff.1 y_real).1.ne',
-        (ENNReal.toReal_pos_iff.1 y_real).2.ne, EReal.bot_lt_coe]
-  · exfalso; exact not_top_lt h
-  · simp only [(ENNReal.toReal_pos_iff.1 x_real).1.ne',
-      (ENNReal.toReal_pos_iff.1 x_real).2.ne, if_false]
-    rcases ENNReal.trichotomy y with (rfl | rfl | y_real)
-    · exfalso; rw [← ENNReal.bot_eq_zero] at h; exact not_lt_bot h
-    · simp
-    · simp only [(ENNReal.toReal_pos_iff.1 y_real).1.ne', ↓reduceIte,
-        (ENNReal.toReal_pos_iff.1 y_real).2.ne, EReal.coe_lt_coe_iff]
-      apply Real.log_lt_log x_real
-      exact (ENNReal.toReal_lt_toReal (ENNReal.toReal_pos_iff.1 x_real).2.ne
-        (ENNReal.toReal_pos_iff.1 y_real).2.ne).2 h
+  split_ifs <;> simp_all
+  guard_target =ₛ Real.log x.toReal < Real.log y.toReal
```

Comment:
I'm not sure whether we should accept non-closing `simp`s like here. I'm trying to minimize the damage by using `guard_target`.

Reply:
Can't you just turn this into a `have` statement at the top and make the simp final?

Reply:
I've added enough arguments to `simp_all` to close the goal.